### DISCUSSION TITLE
sofar_lsw3.yaml can't show "Total Energy Bought"

### DISCUSSION THREAD

**Discussion by megig** [StephanJoubert/home_assistant_solarman#155] [Q&A]
Dear all,
I install success but can't show Total Energy Bought
Help me
[https://postimg.cc/gallery/9XJ0Pzj](url)

**Comment by megig** [comment]
<a href='https://postimg.cc/2LfX433w' target='_blank'><img src='https://i.postimg.cc/2LfX433w/Screenshot-2565-12-18-at-12-07-24.png' border='0' alt='Screenshot-2565-12-18-at-12-07-24'/></a>

**Comment by hyneksuchodol** [comment]
Hi, I have a same problem with Deye SUN-3K-G (string inverter). Communication is OK, but this value is missing. But the inverter must know it when it sends it to the SolarmanSMART App. Could someone help? Maybe @StephanJoubert ?
